.babelrc

```diff
@@ -1,3 +1,12 @@
 {
-  "presets": ["es2015"]
+  "presets": [
```

Comment:
Without these changes, `npm install` would print:
```
🙌 Thanks for using Babel: we recommend using babel-preset-env now:
please read babeljs.io/env to update!
```
https://babeljs.io/docs/en/env/#before-you-do-anything